docs/getting-started/integrations/vuejs-v3.md

```diff
@@ -88,49 +105,54 @@ export default {
 			editor: ClassicEditor,
 			editorData: '<p>Hello from CKEditor 5 in Vue!</p>',
 			editorConfig: {
-			  plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
-			  toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
-			  licenseKey: '<YOUR_LICENSE_KEY>',
-			  mention: { 
-				  // Mention configuration
-			  },
-			},
+				plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+				toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				licenseKey: '<YOUR_LICENSE_KEY>',
+				// Other configuration options...
+			}
 		};
 	}
 }
 </script>
 ```
 
-### Using the component locally
+#### Using the `<ckeditor>` component globally
 
-If you do not want the CKEditor&nbsp;5 component to be enabled globally, you can skip the `use()` part entirely. Instead, configure it in the `components` property of your view.
+If you do not want to enable the CKEditor&nbsp;5 component globally, you can import the `Ckeditor` component from the `@ckeditor/ckeditor5-vue` package directly into the Vue component where you want to use it, and add it to the `components` object.
```

Comment:
Is the `Ckeditor`correct here? It is different than any other instance in this part.

Reply:
Yep, see: https://cksource.slack.com/archives/C062QP59Y30/p1721033139466469

Reply:
OK, thanks.